pkg/acceptance/helpers/external_volume_client.go

```diff
@@ -21,39 +20,56 @@ func NewExternalVolumeClient(context *TestClientContext, idsGenerator *IdsGenera
 	}
 }
 
-func (c *ExternalVolumeClient) exec(sql string) error {
-	ctx := context.Background()
-	_, err := c.context.client.ExecForTests(ctx, sql)
-	return err
+func (c *ExternalVolumeClient) client() sdk.ExternalVolumes {
+	return c.context.client.ExternalVolumes
 }
 
-// TODO(SNOW-999142): Use SDK implementation for External Volume once it's available
+// TODO switch to returning *sdk.ExternalVolume
+// need to update existing acceptance tests for this
 func (c *ExternalVolumeClient) Create(t *testing.T) (sdk.AccountObjectIdentifier, func()) {
```

Comment:
Haven’t run integration tests for other tests that use this `Create` method. I don't actually use it in my tests so we could leave it as is, but nice to fix it up if possible. I've still left the return type the same to avoid having to refactor too much on this PR.